src/blocks/blocks/form/edit.js

```diff
@@ -651,7 +651,7 @@ const Edit = ({
 			} else {
 				createNotice(
 					'error',
-					__( 'An error has occurred: ', 'otter-blocks' ) + ( res?.error || __( 'unknown', 'otter-blocks' ) ),
+					__( 'An error has occurred:', 'otter-blocks' ) + ( res?.error || __( 'unknown', 'otter-blocks' ) ),
```

Comment:
The space there is intentional so the error message has a space between the previous sentence. Either we can add it back or switch to using sprintf instead.